### PR TITLE
Video Container Fix

### PR DESCRIPTION
**Pull request by MatkoTom**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant
motivation and context. List any dependencies that are required for this change.

Fixed an issue where video container would be to wide when dealing with portrait videos. Also, changed how image resiting works and added some fallback if thumbnail is broken.

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can
reproduce. Please also list any relevant details for your test configuration. All testing was done
manually by following, plainly, functionality requirements. If there are some special edge cases or
special ways that things need to be tested they will be mentioned below.

Dev testing

**Test Configuration**:

* Firmware version:
* Hardware: SAMSUNG Galaxy S22
* SDK: Android 14

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] Any dependent changes have been merged and published in downstream modules

BEFORE: 
JPEG_20240118_142946_848849399047877862

AFTER:
JPEG_20240118_142946_7701187842140388256

### CODE REVIEW

**Review by ivolarevic — APPROVED** [review]
LGTM

File: app/src/main/java/com/clover/studio/spikamessenger/ui/main/chat/ChatAdapter.kt [hunk 1]
```diff
@@ -979,6 +970,22 @@ class ChatAdapter(
         }
     }
 
+    private fun handleMediaResize(body: MessageBody?): Pair<Int, Int> {
+        val imageResized: Pair<Int, Int> = if (body?.thumb != null) {
```
[the remark this answers is not in the document]
Comment (MatkoTom) [reply app/src/main/java/com/clover/studio/spikamessenger/ui/main/chat/ChatAdapter.kt]: Sure

**Review by ivolarevic — APPROVED** [review]
LGTM
